src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ServiceBusConfiguration.java

```diff
@@ -1,98 +1,47 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.config;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.microsoft.azure.servicebus.IMessageHandler;
 import com.microsoft.azure.servicebus.IQueueClient;
-import com.microsoft.azure.servicebus.MessageHandlerOptions;
-import com.microsoft.azure.servicebus.QueueClient;
-import com.microsoft.azure.servicebus.ReceiveMode;
-import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
-import com.microsoft.azure.servicebus.primitives.ServiceBusException;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Lazy;
-import uk.gov.hmcts.reform.bulkscanprocessor.services.ErrorNotificationService;
+import org.springframework.context.annotation.Profile;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.MessageAutoCompletor;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.ServiceBusHelper;
-import uk.gov.hmcts.reform.bulkscanprocessor.tasks.ErrorNotificationHandler;
-
-import java.time.Duration;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import javax.annotation.PostConstruct;
 
 @Lazy
```

Comment:
I think `@Lazy` was so clients won't get initialised until called. Since connection string (evaluations) has been moved to `QueueClientConfig`, perhaps this annotation has to be moved there too? Or left here but added to `QueueClientConfig` as well

Reply:
Or maybe we should just remove it from here? The creation of QueueClientConfig's beans depends on the active Spring profile.

Reply:
I suppose this one doesn't really matter since `QueueClientConfig` will be evaluated on boot so yes - can be removed ¯\\\_(ツ)_/¯